.github/workflows/test.yml

```diff
@@ -3,11 +3,11 @@ on:
   push:
     branches:
       - main
-      - release-0.4.x
+      - release/0.4.x
```

Comment:
can we delete [this branch](https://github.com/holochain/lair/tree/release-v0.4.4) then?